tools/cli2md/cli2md.go

```diff
@@ -42,8 +42,8 @@ func emitFlags(w io.Writer, flags []*kingpin.FlagModel) {
 		return
 	}
 
-	fmt.Fprintf(w, "| Flag | Short | Default | Help |\n")
-	fmt.Fprintf(w, "| ---- | ----- | --- | --- |\n")
+	fmt.Fprintf(w, "| Flag | Short | Default | Help |\n") //nolint:errcheck
```

Comment:
Wondering, given that there are many `fmt.Fprint*` and `fmt.Print*` where the errors are ignored, would it make sense to create a helper function that ignores the error and is used in cases were it makes sense to ignore the error, or maybe there's a separate helper that checks and panics?
WDYT?

Reply:
honestly not worth doing it, if we can't write to stdout, panic is not something anybody's going to see anyway. I think in such case `SIGPIPE` will kill the process anyway.

Reply:
It depends, doesn't it?

- In the case of `fmt.Print*`, if `stdout` is closed (e.g., a pipe was closed), `stderr` _may_ still be available and by default `panic`'s output go to `stderr`.
- In the case of `fmt.Fprint*`, the output may be a file or something other than `stdout` (it could be `stdout` as well). The panic output would go to `stderr`.